src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -1407,12 +1407,12 @@ public ResponseEntity<Void> copyVoltageInitModifications(@PathVariable("studyUui
         return ResponseEntity.ok().build();
     }
 
-    @GetMapping(value = "/up-optional-services")
-    @Operation(summary = "Get all the available optional services")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "List of available optional services")})
-    public ResponseEntity<List<String>> getUpOptionalServices() {
-        List<String> upServices = actuatorHealthService.getUpOptionalServices();
-        return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(upServices);
+    @GetMapping(value = "/optional-services")
+    @Operation(summary = "Get all the optional services and their status")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "List of optional services")})
+    public ResponseEntity<String> getOptionalServices() {
+        String optionalServicesStatus = actuatorHealthService.getOptionalServices();
```

Comment:
Why you do not use a DTO ?

Reply:
well I dont see how to make the excepted Json using DTO...
Easy If I change the Json into:
```
[
  {
    "service": "security-analysis-server",
    "status": "up"
  },
  {
    "service": "short-circuit-server",
    "status": "down"
  },
 ...   
]
```